apps/damap-frontend/src/app/components/layout/layout.component.html

```diff
@@ -134,7 +134,7 @@
     </div>
   </mat-sidenav>
   <mat-sidenav-content class="mat-sidenav-content">
-    <app-env-banner *ngIf="env === 'DEV'"></app-env-banner>
+    <!-- <app-env-banner *ngIf="env === 'DEV'"></app-env-banner> -->
```

Comment:
Should this really be removed? It provides information for users on the test instance.